tests/unit/test-util/test.file-filter.js

```diff
@@ -103,22 +113,14 @@ describe('util/file-filter', () => {
 
   });
 
-  describe('normalizeResolve', () => {
-    const paths = [
-      'file', 'dir/',
-      'path/to/file', 'path/to/dir/', 'path/to/../file', 'path/to/../dir/',
-      'path/to/dir/.', 'path/to/dir/..',
-    ];
-
-    it('mimic path.resolve', () => {
-      const src = '/src/';
-
-      paths.forEach((file) => {
-        assert.equal(
-          path.resolve(src, file),
-          path.join(path.resolve(src), normalizeResolve(file))
-        );
-      });
+  describe('isSubDir', () => {
+    it('test if target is a sub directory of src', () => {
+      assert.equal(isSubDir('dist', '.'), false);
+      assert.equal(isSubDir('.', 'artifacts'), true);
+      assert.equal(isSubDir('.', '.'), false);
+      assert.equal(isSubDir('/src/dist', '/src'), false);
+      assert.equal(isSubDir('/src', '/src/artifacts'), true);
+      assert.equal(isSubDir('/src', '/src'), false);
```

Comment:
Can you add one more test to check that two completely unrelated paths return the right value? This is passing but I think it will be good to cover it.
````diff
diff --git a/tests/unit/test-util/test.file-filter.js b/tests/unit/test-util/test.file-filter.js
index a939c5d..0ee8e41 100644
--- a/tests/unit/test-util/test.file-filter.js
+++ b/tests/unit/test-util/test.file-filter.js
@@ -121,6 +121,7 @@ describe('util/file-filter', () => {
       assert.equal(isSubDir('/src/dist', '/src'), false);
       assert.equal(isSubDir('/src', '/src/artifacts'), true);
       assert.equal(isSubDir('/src', '/src'), false);
+      assert.equal(isSubDir('/firstroot', '/secondroot'), false);
     });
   });
 

````